src/ReactiveUI.Tests/API/ApiApprovalTests.ReactiveUI.netcoreapp3.1.approved.txt

```diff
@@ -257,6 +257,8 @@ namespace ReactiveUI
         ReactiveUI.BindingDirection Direction { get; }
         TView View { get; }
         System.Linq.Expressions.Expression ViewExpression { get; }
+        [System.Obsolete("This property has been deprecated. Refer to ViewModelExpression for a representat" +
```

Comment:
representat is a spelling mistake I suspect. 

`Refer to ViewModelExpression for a way to retrieve the value.` maybe.

Reply:
Hmm looks correct on the declaration but the approval text gets messed up. I'll correct it. Supposed to be "This property has been deprecated. Refer to ViewModelExpression for a representation of the bound view model."

Reply:
Ah if it's just a approval text thing then don't worry about it.